packages/domain/lib/use_case/get_people_use_case.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:domain/model/cast.dart';
+import 'package:domain/model/people_and_images_model.dart';
+import 'package:domain/model/people_response.dart';
+import 'package:domain/model/tmdb_response.dart';
+import 'package:domain/repository/tmdb_repository.dart';
+import 'package:domain/repository/trakt_repository.dart';
+import 'package:domain/use_case/use_case.dart';
+import 'package:domain/utils/const.dart';
+
+class GetPeopleUseCase
+    extends UseCaseParams<String, Future<List<PeopleAndImagesModel>>> {
+  GetPeopleUseCase(
+    this._generalRepository,
+    this._tMDBRepository,
+  );
+
+  final TRAKTRepository _generalRepository;
+  final TMDBRepository _tMDBRepository;
```

Comment:
I choose this naming because if I do your way linter underscore this names:

<img width="246" alt="Screenshot 2022-09-30 at 16 57 53" src="https://user-images.githubusercontent.com/111364147/193285988-dfd23854-10e1-45dd-a73a-aee9c70b381b.png">